synapse/lib/stormtypes.py

```diff
@@ -5300,6 +5300,11 @@ class Layer(Prim):
             ''',
          'type': {'type': 'function', '_funcname': 'getStorNodes',
                   'returns': {'name': 'Yields', 'type': 'list', 'desc': 'Tuple of buid, sode values.', }}},
+        {'name': 'getMirrorStatus', 'desc': '''
```

Comment:
Not related to getMirrorStatus, but probably also want to sanitize the mirror url for pack()?